File: src/main/java/com/github/ccguyka/showupdates/filter/pom/Project.java
package com.github.ccguyka.showupdates.filter.pom;

import java.util.ArrayList;
import java.util.List;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Project {

    private List<Dependency> dependencies = new ArrayList<>();
    private DependencyManagement dependencyManagement = new DependencyManagement();
    private Build build = new Build();

    public List<Dependency> getDependencies() {
        return dependencies;
    }

    public void setDependencies(List<Dependency> dependencies) {
        this.dependencies = dependencies;
    }

    public DependencyManagement getDependencyManagement() {
        return dependencyManagement;
    }

    public void setDependencyManagement(DependencyManagement dependencyManagement) {
        this.dependencyManagement = dependencyManagement;
    }

    public Build getBuild() {
        return build;
    }

    public void setBuild(Build build) {
        this.build = build;
    }
}

File: src/main/java/com/github/ccguyka/showupdates/filter/pom/Project.kt
package com.github.ccguyka.showupdates.filter.pom

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.util.*

@JsonIgnoreProperties(ignoreUnknown = true)
class Project {

    var dependencies: List<Dependency> = ArrayList()
    var dependencyManagement = DependencyManagement()
    var build = Build()
}
